.pullapprove.yml

```
@@ -127,8 +127,7 @@ groups:
           'packages/animations/**/{*,.*}',
           'packages/platform-browser/animations/**/{*,.*}',
           'aio/content/guide/animations.md',
-          'aio/content/guide/animate-state-style.md',
-          'aio/content/guide/animation-api-summary.md',
+          'aio/content/guide/add-an-animation.md',
```

It looks like this overwrote 2 pre-existing file paths that you'll need to add back in.
```suggestion
          'aio/content/guide/animate-state-style.md',
          'aio/content/guide/animation-api-summary.md',
          'aio/content/guide/add-an-animation.md',
```

---

Sorry for all the back and forth on this. I know this can be frustrating.